ui/models/submission.md

```diff
@@ -0,0 +1,68 @@
+# Submission
+
+The top level object encapsulating a logical grouping of data and metadata to be deposited to the data archive. A submission will contain one or more files, with associated file metadata, that a user intends to submit to the data archive. The user must also provide metadata describing the submission as a whole. Much of the submission data will be provided by the user throught the UI
```

Comment:
This is a nit picky thing and a style point, but you might consider adding periods for the closing sentence in many of the table cells where there are two or more sentences.